tests/ModelTest.php

```diff
@@ -834,12 +834,12 @@ public function testChunkById(): void
         User::create(['name' => 'spork', 'tags' => ['sharp', 'pointy', 'round', 'bowl']]);
         User::create(['name' => 'spoon', 'tags' => ['round', 'bowl']]);
 
-        $count = 0;
-        User::chunkById(2, function (EloquentCollection $items) use (&$count) {
-            $count += count($items);
+        $names = [];
+        User::chunkById(2, function (EloquentCollection $items) use (&$names) {
+            $names = array_merge($names, $items->pluck('name')->all());
```

Comment:
Would it make sense to use `+=` here for an [array union](https://www.php.net/manual/en/language.operators.array.php)? I think it'd have the same effect for a list, and we can safely assume these aren't associative arrays.

If you like, you could also add a duplicate name to the fixtures above to assert that it's appended instead of merged. I'll note that there is no such problem with `array_merge()` as written here (again, since lists are concatenated).

Reply:
The `+` doesn't work well with lists, as it behave like `array_replace`.

```php
var_dump(['a', 'b'] + ['c']);
// Result: ['a', 'b']
```